scripts/cleanup.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+updateControllerClusterStatus() {
+    echo "🔧 Updating registration status"
+    kubectl patch clusters.controller.kubeslice.io $CLUSTER_NAME -n $PROJECT_NAMESPACE --token $TOKEN --certificate-authority /ca.crt --server $HUB_ENDPOINT --type=merge --subresource status --patch "{\"status\": {\"registrationStatus\": \"$1\"}}"
+}
+
+removeFinalizer() {
+    echo "🗑 Removing cluster deregister finalizer"
+    kubectl get -o yaml clusters.controller.kubeslice.io $CLUSTER_NAME -n $PROJECT_NAMESPACE --token $TOKEN  --certificate-authority /ca.crt --server $HUB_ENDPOINT > ./cluster.yaml
+    sed -i '/finalizers:/,/^[^ ]/ s/ *- networking.kubeslice.io\/cluster-deregister-finalizer//' cluster.yaml
+    kubectl patch clusters.controller.kubeslice.io $CLUSTER_NAME -n $PROJECT_NAMESPACE --type=merge  --patch-file cluster.yaml --token $TOKEN  --certificate-authority /ca.crt --server $HUB_ENDPOINT
+}
+
+deleteKubeSliceCRDs() {
+    for item in $(kubectl get crd | grep "networking.kubeslice.io"); do
+    echo "🗑 Removing item $item"
+    kubectl delete crd $item --ignore-not-found
+    done
+}
+
+deleteNamespace(){
+    echo "🗑 Removing kubeslice-system namespace"
+    kubectl delete namespace kubeslice-system
+}
+
+SECRET=kubeslice-hub
+TOKEN=$(kubectl get secret ${SECRET} -o json | jq -Mr '.data.token' | base64 -d)
+kubectl get secret ${SECRET} -o json | jq -Mr '.data["ca.crt"]' | base64 -d > /ca.crt
+# get the worker release name
+workerRelease=$(helm list --output json -n kubeslice-system | jq -r '.[] | select(.chart | startswith("kubeslice-worker-")).name')
+echo "🏃 Running helm uninstall for the release $workerRelease" 
+if helm uninstall $workerRelease --namespace kubeslice-system
+then
+    # delete crds
+    deleteKubeSliceCRDs
```

Comment:
Can we please move all these commands into a func and reuse it in the else clause below?

if helm_uninstall_succeeded
then
    run_post_uninstall_cleanup
else
   if helm_uninstall_no_hook_succeeded
   then
       run_forced_post_uninstall_cleanup


run_forced_post_uninstall_cleanup(){
    # Do forced ops here
    run_post_uninstall_cleanup
}